Makefile

```diff
@@ -32,15 +32,13 @@ lint: clean
 	pipenv run flake8 bwapi tests --exit-zero
 
 ## run tests with the default Python
-test:
-	pipenv run pytest
+test: lint
+	pipenv run pytest -vv --cov=bwapi
 
 ## check code coverage quickly with the default Python
-coverage:
-	pipenv run coverage run --source bwapi -m pytest
-	pipenv run coverage report -m
-	pipenv run coverage html
-
+coverage: clean
```

Comment:
NABD but I noticed that `coverage` target isn't marked as `.PHONY`. Neither is `lint` or `upload` come to think of it. Obviously this isn't to do with your changes, but since you're on a clean-up kick I thought you'd like to know 😉